app/views/pages/induction_tutor_materials/ambition/year_two.html.erb

```diff
@@ -0,0 +1,110 @@
+<% content_for :before_content, govuk_back_link(text: 'Back', href: current_user ? dashboard_path : root_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-9">Ambition Institute's handbooks and training outlines</h1>
+
+    <h2 class="govuk-heading-l govuk-!-margin-bottom-3">Handbooks</h2>
+
+    <ul class="govuk-list govuk-list--spaced">
+      <li>
+        <%= govuk_link_to "ECF Programme Handbook (PDF, 107 KB, 10 pages)",
+                          href = "https://www.early-career-framework.education.gov.uk/ambition/wp-content/uploads/sites/3/2020/08/Ambition-EarlyCareerTeachers_2020_ProgrammeHandbook.pdf" %>
```

Comment:
Do we need to say `href=`? I thought it would be enough to say 
```
<%= govuk_link_to "ECF Programme Handbook (PDF, 107 KB, 10 pages)", "https://www.early-career-frame... %>"
```